068-performance-testing.md

```diff
@@ -0,0 +1,153 @@
+
+#### Note on Release Lifecycle:
+
+Embedding performance tests within the strimzi-kafka-operator's system test module means performance
+testing phases are tightly coupled with the operator's release cycle.
+This could streamline the release process but might also necessitate additional considerations for test maintenance
+and updates aligned with each release.
+
+---
+
+Nevertheless, this approach limits the applicability of the performance tests to the strimzi-kafka-operator, excluding potential integration
+with other repositories such as strimzi-kafka-bridge and strimzi-kafka-oauth, leading us to consider a second approach.
+
+### Approach 2: Creating a Dedicated Repository for Performance testing
+
+This method proposes creating a separate, dedicated repository for performance testing across the Strimzi ecosystem,
+addressing some challenges of the first approach while introducing new considerations.
+A distinct advantage of this strategy is its facilitation of integration with various Strimzi projects,
+enhancing the flexibility and reach of performance testing.
+
+However, this approach requires re-implementing foundational components such as ResourceManager, SetupClusterOperator, LogCollector, and MetricsCollector.
+While this might seem daunting, leveraging a framework like [Test Frame](https://github.com/skodjob/test-frame) could streamline the process.
+Test Frame provides a robust starting point with essential classes that can be easily extended to fit our needs,
+such as developing a TopicMetricsCollector by extending the MetricsCollector class.
+
+Despite these benefits, integrating this dedicated repository with the strimzi-kafka-operator poses a more complex challenge.
+Achieving seamless integration requires careful planning and coordination, ensuring that performance tests can be effectively
+executed and monitored across different components of the Strimzi ecosystem.
+
+#### Note on Release Lifecycle:
+
+A dedicated repository for performance testing introduces a separate release lifecycle for the testing framework itself.
+This separation offers flexibility in testing across different versions of Strimzi components but requires careful
+synchronization to ensure compatibility and relevance of performance tests to the versions of Strimzi components being tested.
+It also allows for independent evolution of the testing framework, accommodating new testing needs and methodologies
+without being constrained by the release cycles of the Strimzi components it tests.
+
+---
+
+This approach may appear more effective than the initial one, yet it necessitates the creation of numerous support classes for:
+
+1. Extracting metrics from Kubernetes Pods components
+2. Monitoring the entire lifecycle
+3. Documenting (i.e., reporting) the outcomes of performance tests
+    4. this could all metrics scraped during performance test
+    5. and then reporting via table of each parameter monitored during such test
```

Comment:
```suggestion
4. this could all metrics scraped during performance test
5. and then reporting via table of each parameter monitored during such test
```